bindings/rust/standard/s2n-tls-hyper/Cargo.toml

```diff
@@ -14,8 +14,8 @@ default = []
 [dependencies]
 s2n-tls = { version = "=0.3.9", path = "../../extended/s2n-tls" }
 s2n-tls-tokio = { version = "=0.3.9", path = "../../extended/s2n-tls-tokio" }
-hyper = { version = "1" }
-hyper-util = { version = "0.1", features = ["client-legacy", "tokio", "http1", "http2"] }
+hyper = { version = "1.4" }
```

Comment:
hyper 1.4 is needed because [hyper-util 0.1.7 requires it](https://github.com/hyperium/hyper-util/blob/31708072ed06a72cff80972beb7fb52b661b5982/Cargo.toml#L21).